tests/e2e/tutorials/tutorialBase.js

```diff
@@ -227,6 +236,28 @@ class TutorialBase {
     return resp;
   }
 
+  async startSim4LifeLight() {
+    await this.takeScreenshot("startSim4LifeLight_before");
+    this.__responsesQueue.addResponseListener("projects?from_study=");
+    this.__responsesQueue.addResponseListener(":open");
+    let resp = null;
+    try {
+      await this.waitFor(2000);
+      await auto.dashboardStartSim4LifeLight(this.__page);
+      await this.__responsesQueue.waitUntilResponse("projects?from_study=");
+      resp = await this.__responsesQueue.waitUntilResponse(":open");
+      const studyId = resp["data"]["uuid"];
+      console.log("Study ID:", studyId);
+    }
+    catch (err) {
+      console.error(`Sim4Life Light could not be started:\n`, err);
+      throw (err);
+    }
+    await this.waitFor(2000);
+    await this.takeScreenshot("startSim4LifeLight_after");
```

Comment:
Q: how this tutorial differs from the normal S4L tutorial? Besides the names of the screenshots?
Isn't it identical services but you target a different url?

Reply:
Different ways to start the same service and different endpoint checks and, for now:
- In the current S4L tutorials, we start the sim4life service by clicking the service card in the services tab. sim4life light does it by clicking the plus button in the projects tab. It doesn't even expose the services tab.
- In this new test we also make sure that every service provided in the ```/services``` endpoint has the ```simcore/services/dynamic/sim4life-dy``` key
- Once the service is started, the interaction is identical.